rlclientlib/serialization/fb_serializer.h

```diff
@@ -35,9 +35,28 @@ namespace reinforcement_learning { namespace logger {
       TimeStamp client_ts(	ts.year, ts.month, ts.day, ts.hour,
 							ts.minute, ts.second, ts.sub_second);
 	    const auto meta_id_offset = CreateMetadata(builder,&client_ts);
+
+      flatbuffers::Offset<DecisionMode> decision_mode_offset;
+      switch (evt.get_decision_mode()) {
+      case IMITATION_MODE: {
+        const auto imitation_mode = CreateImitationMode(builder).Union();
+        const auto imitation_mode_name = builder.CreateString("imitation_mode");
+        decision_mode_offset = CreateDecisionMode(builder, imitation_mode_name, Mode_ImitationMode, imitation_mode);
+        break;
+      }
+      case ONLINE_MODE: {
+        const auto default_mode = CreateOnlineMode(builder).Union();
+        const auto default_mode_name = builder.CreateString("online_mode");
+        decision_mode_offset = CreateDecisionMode(builder, default_mode_name, Mode_OnlineMode, default_mode);
+        break;
+      }
+      default:
+        break;
```

Comment:
Is this not an error?

Reply:
No. It's not.

This is to be back-compatible with the config not setting learning mode. I will add a comment in the next push.